### PR TITLE
[FIX] Removed cursor-pointer and small text  when hover the balance

### PR DESCRIPTION
**Pull request by pedropaulodf**
# Description

I removed cursor-pointer and small text  when hover the balance. Only tailwind changes.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

This is only visual changes.

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Pranav-yadav** [comment]
@pedropaulodf are u sure these changes will be proper. Cause, they were kept intentionally.

By default the balance shown is not the accurate balance but the rounded balance, hence it was intentionally kept that way to show the true balance onhover and pointer was changed to let players know that they will see change onhover.

File: src/features/hud/components/Balance.tsx
Comment (Pranav-yadav) [inline src/features/hud/components/Balance.tsx:29]: This was also intentional.

File: src/features/hud/components/Balance.tsx
Comment (Pranav-yadav) [inline src/features/hud/components/Balance.tsx:17]: This was intentional.

**Review by Pranav-yadav — CHANGES_REQUESTED** [review]
Go through the comment in the discussion 👍

**Comment by pedropaulodf** [comment]
Hi Pranav! Thank you for the reponse and the greetings! 😃  I want contribute even it is litle bit.

About the PR: i'm not knew about this intentional implementations. When i saw the behavior of the balance button, i tought it is an error. hahaha

By the way, i have a sugestion: What you think about an tooltip, informing this rounded/real value? You think this tooltip will pollute the design?
---
About other PR that i want to implement: it was talked about the custom quantity crops the user can sell? Like a input for this custom quantity?

**Comment by Pranav-yadav** [comment]
> What you think about remove the small text on hover and add an tooltip, informing this rounded/real value? You think this tooltip will pollute the design?

Actually small text was intentional as well to no overflow too much on the mobile devices as, the balanve can of 18 dev as well ig. 😊


---
About the custom amount input for sell, it has been discussed in dev's chat earlier and we decided to keep it simple & light for the moment. There are plans to go that way, but not just now as per Adam's opinion. 👍

---
If you want, you can close the PR. Else, wait for the review from the OG maintainers. 👍

**Comment by pedropaulodf** [comment]
Ok, i understand. haha
I came late to help de dev this project, so a don't know all de discussions. I will try keep on eye in the discord group.
However, thank you very much Pranav! 

I will close it. Thank you!

**Pull request closed without merging**
